packages/base/src/features/F6Navigation.ts

```diff
@@ -132,6 +133,14 @@ class F6Navigation {
 	}
 
 	async _keydownHandler(event: KeyboardEvent) {
+		const openUI5Support = getFeature<typeof OpenUI5Support>("OpenUI5Support");
+		const isOpenUI5Loaded = openUI5Support ? openUI5Support.isOpenUI5Detected() : false;
```

Comment:
`const isOpenUI5Detected = openUI5Support && openUI5Support.isOpenUI5Detected()`

would probably be a bit more accurate, because in OpenUI5 support we distinguish generally between detected (meaning it's on the page) and loaded (OpenUI5 core has booted)